applications/views.py

```diff
@@ -94,3 +94,27 @@ def patch(self, request, *args, **kwargs):
         set_submission_review_status(submission, review_status)
 
         return Response(status=status.HTTP_200_OK)
+
+
+class UploadResumeView(GenericAPIView):
+    """
+    View for uploading resume and linkedin link
+    """
+    authentication_classes = (SessionAuthentication,)
+    permission_classes = (IsAuthenticated, UserIsOwnerPermission,)
+    lookup_field = "pk"
+    owner_field = "user"
+    queryset = BootcampApplication.objects.all()
+
+    def post(self, request, *args, **kwargs):
```

Comment:
I think the request should be a `PATCH` rather than a `POST`

Reply:
in the django documentation it said that for uploading a file is has to be POST resquest.